container/secrets.go

```diff
@@ -81,5 +81,13 @@ func readFile(root, name string) ([]SecretData, error) {
 }
 
 func getHostSecretData() ([]SecretData, error) {
-	return readAll("/usr/share/rhel/secrets", "")
+	baseSecrets, err := readAll("/usr/share/rhel/secrets", "")
+	if err != nil {
+		return nil, err
+	}
+	overrideSecrets, err := readAll("/etc/container/rhel/secrets", "")
```

Comment:
I think we wanted just /etc/containers/secrets also, no need to hard code rhel.
/etc/containers/secrets
Note plural, which I think we are using for /var/lib/containers

@cgwalters WDYT